### PR TITLE
OV-272: Add logo

### PR DESCRIPTION
**Pull request by stefano-lacorazza**
image
image
image

### CODE REVIEW

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Please create Logo.tsx component and reuse it

File: frontend/src/bundles/auth/components/common/form-header/form-header.tsx [hunk 1]
```diff
@@ -10,8 +12,10 @@ type Properties = {
 const FormHeader: React.FC<Properties> = ({ headerText, subheader }) => {
     return (
         <>
-            {/* TODO: Add logo */}
-            <h2 className={styles['logo']}>LOGO</h2>
+            <h2 className={styles['logo']}>
+                <img src={logo} alt="Logo" />
```
Comment (o-nedashkivska) [inline frontend/src/bundles/auth/components/common/form-header/form-header.tsx:16]: Maybe we can use Chakra UI image component?

File: frontend/src/assets/img/logo-sm.svg
Comment (nikita-remeslov) [inline frontend/src/assets/img/logo-sm.svg:1]: we don't nee it, logo should be responsive

File: frontend/src/bundles/common/components/logo/logo-text.tsx
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/logo/logo-text.tsx:1]: combine it in one component and add some prop to show or hide logo text

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Review by o-nedashkivska — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**